firmware/hackrf_usb/m0_state.c

```diff
@@ -1,6 +1,5 @@
 /*
```

Comment:
Can you please rename this to `usb_api_m0_state.c` for consistency with the other files that implement USB vendor requests?

Reply:
Done